tests/network/services.go

```diff
@@ -32,14 +32,24 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/net"
 
 	v1 "kubevirt.io/client-go/api/v1"
 	"kubevirt.io/client-go/kubecli"
 	"kubevirt.io/kubevirt/tests"
-	"kubevirt.io/kubevirt/tests/flags"
 	"kubevirt.io/kubevirt/tests/libvmi"
 )
 
+type vmiServiceExpositionManager struct {
```

Comment:
I find "Exposition" strange here. Is this related to "Expose"?

Reply:
Yes. That's what a `Service` does. Exposes stuff.

I could only come up with 'exposition' or 'exposure'.

Do you suggest an alternate name for this entity, considering it 'assures all IPs on a VMI primary interface are exposed via services' ?

Reply:
Done; went with `vmiServiceManager`.